pywr/recorders/recorders.py

```diff
@@ -473,6 +473,33 @@ def setup(self):
                         entry[s.name] = i
                     entry.append()
                 tbl.flush()
+            elif any([s.slice for s in self.model.scenarios.scenarios]):
+                # Slices are only applied in a model run if there are no user combinations
+                description = {
+                    "name": tables.StringCol(1024),
+                    "slice_start": tables.Int64Col(),
```

Comment:
I think the "slice_" prefix is redundant here as we're in the context of "scenario slices" for the table.